src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <div class="calendar calendar--start-day--sun calendar--month-length--29">
```

Comment:
Change the calendar modifiers to use double hyphen instead of a single hyphen. For example, 'calendar--start-day--sun' should be 'calendar--start-day-sun'.